pkg/openstack/loadbalancer.go

```diff
@@ -129,7 +130,11 @@ type servicePatcher struct {
 	updated *corev1.Service
 }
 
-var _ cloudprovider.LoadBalancer = &LbaasV2{}
+var (
+	// List based on Octavia documentation https://docs.openstack.org/api-ref/load-balancer/v2/index.html?expanded=create-pool-detail#protocol-combinations-listener-pool
+	proxyProtocolSupportedListeners                            = []listeners.Protocol{listeners.ProtocolHTTP, listeners.ProtocolHTTPS, listeners.ProtocolTCP, listeners.ProtocolTerminatedHTTPS}
```

Comment:
it's better to define each supported protocols on a new line. further list updates will look more readable.